src/converge.js

```diff
@@ -5,10 +6,9 @@ var pluck = require('./pluck');
 
 
 /**
- * Accepts at least three functions and returns a new function. When invoked, this new
- * function will invoke the first function, `after`, passing as its arguments the
- * results of invoking the subsequent functions with whatever arguments are passed to
- * the new function.
+ * Accepts a converging functiona and a list of branching functions and returns a new function.  * When invoked, this new function will invoke the first function, `after`, passing as its
```

Comment:
This line should be wrapped. Ideally the whole paragraph would be wrapped at 79 characters. :)